front/src/reducers/osrdconf/types.ts

```diff
@@ -56,6 +56,7 @@ export interface OsrdStdcmConfState extends OsrdConfState {
   totalMass?: number;
   totalLength?: number;
   maxSpeed?: number;
+  towedRollingStockID?: number;
```

Comment:
Why is the rolling stock ID not present in this interface?

Reply:
This struct extend OsrdConfState that has the rolling stock ID